### PR TITLE
[T03] Create the CLASS creation algorithm

### PR DESCRIPTION
**Pull request by absirgue**
needs commenting and adding in to nice spot when T24 is merged but the logic is done :)
fox

### CODE REVIEW

**Review by Ali-Alkhars — COMMENTED** [review]
I believe it works perfectly! Brilliant Anton.

I have two little things though:

1. Could you add a comment about the 408 status at the top of app.js?
2. You added a comment "send email". Did you make it a comment because you are waiting on Cecilia?

**Review by Ali-Alkhars — APPROVED** [review]
Yalla habibi

File: queries/courses.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  getAllCourses: function (sql, callback) {
+    const request = new sql.Request();
+
+    request.query("SELECT * FROM Courses", function (err, recordset) {
```
Comment (TunaAtasayar) [inline queries/courses.js:5]: SELECT * FROM COURSES WHERE IsActive = 1
Im adding a comment so u dont forget.

**Review by TunaAtasayar — COMMENTED** [review]
idk if it will make a mess if I approve rn when u need to change some lines - so for now im just commenting

**Review by TunaAtasayar — APPROVED** [review]
lezgo foxy

**Pull request merged**